test/test_playbooks/domain_vars.yml

```diff
@@ -2,7 +2,7 @@
 domain_name: "example.org"
 domain_dns_proxy: "centos7-devel.example.com"
 domain_locations:
-  - 'Default Location'
+  - 'Test Location'
```

Comment:
I'd suspect this for using the Default one because it requires a Smart proxy to be manually setup beforehand?

Reply:
Nope, I think this is only used to hide domain on host creation/update. But, yes, has you said, to run `make record_domain` you have to setup manualy a smart-proxy with dns plugin enabled and name it `centos7-devel.example.com`. No orgs or locs are affected to this proxy on my setup and all tests are ok.
![screenshot_foreman_sp](https://user-images.githubusercontent.com/2952632/50901310-12482780-1418-11e9-80d1-9b5e9c36fcea.png)

We can handle this in two ways:

1. If we use `Default Location` (or `Default Organization`) in tests we must not delete it after tests because recent version of foreman must have at least 1 org/loc.
2. Use trowable Orgs/Locs and clean it up after tests

As you can see I choose the second option, because all other tests I saw was doing things like this. But it can be easily reverted if you want.

Reply:
Is this the reason, you needed to rerecord so much tests?
If yes, feel free to resolve again.